.github/CONTRIBUTING.md

```diff
@@ -77,6 +77,11 @@ ok      github.com/containous/traefik   0.005s  coverage: 4.1% of statements
 Test success
 ```
 
+For dev purpose, you can skip some integration tests by using:
```

Comment:
It's more "you can specifiy which tests to run" instead of skipping :angel: 

```bash
TESTFLAGS="-check.f MyTestSuite.MyTest" make test-integration
```

`MyTest` can even be not the full name, like if if use `-check.f MyTestSuite.My` it will run all tests starting by `My` :wink: